File: Sources/Indigox.UUM.NHibernateImpl/Model/OperationLog.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Indigox.Common.StateContainer;

namespace Indigox.UUM.NHibernateImpl.Model
{
    public class OperationLog
    {
        public int ID { get; set; }
        public string Operator { get; set; }
        public string Operation { get; set; }
        public DateTime OperationTime { get; set; }
        public string DetailInformation { get; set; }
    }
}

File: WebApp/res/js/Indigox/UUM/Application/OperationLog.js
define("Indigox/UUM/Application/OperationLog",
    [
        "Indigox.Web.JsLib.Models.RecordManager"
    ],
function (
    RecordManager
) {
    RecordManager.getInstance().register('OperationLog', {
        columns: [
            { name: 'ID', text: '编号', type: String },
            { name: 'Operator', text: '用户', type: String },
            { name: 'Operation', text: '操作描述', type: String },
            { name: 'OperationTime', text: '操作时间', type: String },
            { name: 'DetailInformation', text: '详细信息', type: String }
        ],
        primaryKey: ['ID']
    });
});